components/MUI/getMUITheme.ts

```diff
@@ -12,18 +12,49 @@ export type ThemeMode = Theme["palette"]["mode"];
 const spacing = 4;
 // TODO: I'm not sure whether it should be 4 or 5
```

Comment:
```suggestion
const spacing = 5;
```

It looks like 5 is the initial convention started by a designer

Can you double-check?